src/popups.rs

```diff
@@ -206,18 +206,19 @@ pub fn render_help_popup(frame: &mut Frame) {
         Line::from("Game controls:".underlined().bold()),
         Line::from(""),
         Line::from(vec![
-            "← ↑ ↓ →: Use the arrow keys to move the ".into(),
+            "←/h ↑/k ↓/j →/l: Use these keys to move the ".into(),
             "blue".blue(),
             " cursor".into(),
         ]),
         Line::from(""),
-        Line::from("Ctrl + or -: Zoom in or out to adjust pieces sizes"),
+        Line::from("`Ctrl` '+' or '-': Zoom in or out to adjust pieces sizes"),
+        Line::from("(Might differ in certain terminals)"),
         Line::from(""),
-        Line::from("SPACE_BAR: Select a piece"),
+        Line::from("`Space`: Select a piece"),
         Line::from(""),
-        Line::from("ESCAPE: Deselect a piece"),
+        Line::from("`Esc`: Deselect a piece, don't display popups"),
```

Comment:
```suggestion
        Line::from("`Esc`: Deselect a piece / hide popups"),
```

Reply:
this is good, I'll implement it right now